silk/tests/test_integration.py

```diff
@@ -0,0 +1,83 @@
+# (C) Datadog, Inc. 2021-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import mock
+import pytest
+
+from datadog_checks.silk import SilkCheck
+
+from .common import BASE_TAGS, BLOCKSIZE_METRICS, HOST, METRICS, READ_WRITE_METRICS, SYSTEM_TAGS
+
+
+@pytest.mark.parametrize(
+    'enable_rw, enable_bs, expected_metrics',
+    [
+        pytest.param(False, False, [*METRICS], id="rw bs disabled"),
+        pytest.param(True, True, [*METRICS, *BLOCKSIZE_METRICS, *READ_WRITE_METRICS], id="rw bs enabled"),
+        pytest.param(False, True, [*METRICS, *BLOCKSIZE_METRICS], id="bs enabled"),
+        pytest.param(True, False, [*METRICS, *READ_WRITE_METRICS], id="rw enabled"),
+    ],
+)
+@pytest.mark.integration
+@pytest.mark.usefixtures('dd_environment')
+def test_check(dd_run_check, aggregator, instance, enable_rw, enable_bs, expected_metrics):
+    instance['enable_read_write_statistics'] = enable_rw
+    instance['enable_blocksize_statistics'] = enable_bs
+
+    check = SilkCheck('silk', {}, [instance])
+    dd_run_check(check)
+    for metric in expected_metrics:
+        aggregator.assert_metric(metric)
+        for tag in [*BASE_TAGS, *SYSTEM_TAGS]:
+            aggregator.assert_metric_has_tag(metric, tag)
```

Comment:
nit - Can't we do something like this?

```suggestion
        aggregator.assert_metric(metric, tags=[*BASE_TAGS, *SYSTEM_TAGS])
```

Or can we have some other tags?

Reply:
There are indeed some other tags that are specific to each metric, but I will leave a note to make it more clear.